packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.tsx

```diff
@@ -314,18 +314,20 @@ const AccountRecoveryResetPassword = ({
     setBannerState(BannerState.PasswordResetSuccess);
   }
 
-  function navigateAway() {
-    setUserPreference('account-recovery', account.recoveryKey);
+  async function navigateAway() {
+    // TODO / refactor: the initial account result with useAccount() does not
```

Comment:
There aren't issues filed for the TODOs in this PR yet, but we will definitely be filing issues out of the settings architecture doc which specifically calls out our API calls in Account, so these will probably be filed whenever the rest of those are.